src/main/java/io/fabric8/maven/docker/util/WaitUtil.java

```diff
@@ -25,6 +25,8 @@
  */
 public class WaitUtil {
 
+    private static final int DEFAULT_DOCKER_TIMEOUT = 10;
```

Comment:
i don't always want to wait - the docker container takes up to 10 seconds which is why i used that default. if it exited before then, the future would complete, so i don't think we were waiting any extra time w/ this.

but whatever you think is best here.